code/modules/wiremod/components/hud/click_intercept.dm

```diff
@@ -19,8 +19,8 @@
 /obj/item/circuit_component/click_intercept/Initialize()
 	. = ..()
 	trigger_input = add_input_port("Activate", PORT_TYPE_SIGNAL)
-	trigger_output = add_output_port("Clicked", PORT_TYPE_SIGNAL)
-	clicked_atom = add_output_port("Clicked Object", PORT_TYPE_ATOM)
+	trigger_output = add_output_port("Target", PORT_TYPE_SIGNAL)
```

Comment:
Call it triggered or something, to indicate it's a signal. Target suggests PORT_TYPE_ATOM